config/tool_destinations.yml.sample

```diff
@@ -121,7 +121,9 @@
 #         destination: fail
 #         fail_message: Don't do that
 #         arguments:
-#           careful: true
+#           my_section:
```

Comment:
For the real world example, I'm not really inspired and a bit lazy :P 
To complexify, I can add a second statement?